view/frontend/templates/onepage.phtml

```diff
@@ -1,8 +1,16 @@
+<?php
+$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+$store = $objectManager->get('Magento\Framework\Locale\Resolver');
+?>
+
 <div id="checkout" class="checkout-container"></div>
 
 <script>
-  var config = <?= \Zend_Json::encode($block->getCheckoutConfig()) ?>;
-  var baseUrl = '<?= $block->getBaseUrl() ?>';
+  var snowdog = snowdog ? snowdog : {};
+
+  snowdog.config = <?= \Zend_Json::encode($block->getCheckoutConfig()) ?>;
+  snowdog.baseUrl = '<?= $block->getBaseUrl() ?>';
+  snowdog.storeLocale = '<?= $store->getLocale() ?>';
```

Comment:
Why did you add one global `snowdog` variables and under it a few nodes?

Reply:
to putt the matter clearly and to know it's variables from our module without mess it up with other m2 global vars